config/v1/types_feature.go

```diff
@@ -105,7 +105,8 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 	TechPreviewNoUpgrade: newDefaultFeatures().toFeatures(),
 	LatencySensitive: newDefaultFeatures().
 		with(
-			"TopologyManager", // sig-pod, sjenning
+			"HugePageStorageMediumSize", // cnf, alukiano
```

Comment:
I would expect this to go under TechPreviewNoUpgrade given the upstream feature is still Alpha.  Having it here implies a greater level of support.

Reply:
Agree with Derek, the hugepages KEP is Alpha. 

https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190129-hugepages.md